entropylab/api/experiment.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, definition: "ExperimentDefinition", db: DataWriter):
         self._experiment_resources: ExperimentResources = (
             definition.get_experiment_resources()
         )
-        self._user: str = ""
+        self._user: str = definition._user
```

Comment:
Maybe check if `definition._user` is `None` and set `self._user` to `""` in that case?